model/app.py

```diff
@@ -218,7 +218,7 @@ def key_from_model_info(wild_type_id, message):
     :param message: dict
     :return: str
     """
-    d = {k: message.get(k, []) for k in {GENOTYPE_CHANGES, MEDIUM, MEASUREMENTS}}
```

Comment:
Alright, because now it appears to be the case that if you add reactions via post, then query the same model again but without added reactions, you'll get the model that had the added reactions.. Revert this change? Or doesn't hurt either no?